### DISCUSSION TITLE
Error in converting Safetensors to GGUF (convert_hf_to_gguf.py)

### DISCUSSION THREAD

**Discussion by lazyperson1020** [ggml-org/llama.cpp#8758] [Q&A]
I got the following error while converting my 16GB model to gguf.
<img width="938" alt="image" src="https://github.com/user-attachments/assets/f1979aca-062d-4e9e-bfe3-4071135c8479">

**Comment by misutoneko** [comment]
Try with ipdb or do some printf debugging I guess?
This trace only shows it's trying to write something with a wrong type, that's all.
The GGUFValueType class is defined in [gguf-py/gguf/constants.py.](https://github.com/ggerganov/llama.cpp/blob/master/gguf-py/gguf/constants.py#L1262)
